### PR TITLE
Removal of test/debug code.

### PR DESCRIPTION
**Pull request by borisroman**
Removed main() method from LocalTemplateDownloader.java

### CODE REVIEW

**Comment by wido** [comment]
LGTM

**Comment by DaanHoogland** [comment]
@borisroman I would like to see all the test-main removed as well but not without moving the actual test code into unit tests

**Comment by borisroman** [comment]
@DaanHoogland True that. But then the discussion arises; until there are Unit Tests , do we keep this useless code?

File: core/src/com/cloud/storage/template/LocalTemplateDownloader.java [hunk 1]
```diff
@@ -159,16 +159,4 @@ public long download(boolean resume, DownloadCompleteCallback callback) {
             }
         }
     }
```
Comment (DaanHoogland) [inline core/src/com/cloud/storage/template/LocalTemplateDownloader.java]: how about a unit test doing
        String url = "file:///home/ahuang/Download/E3921_P5N7A-VM_manual.zip";
        TemplateDownloader td = new LocalTemplateDownloader(null, url, "/tmp/mysql", TemplateDownloader.DEFAULT_MAX_TEMPLATE_SIZE_IN_BYTES, null);
        long bytes = td.download(true, null);
        assertTrue("download failed", bytes > 0);
?

Comment (borisroman) [inline core/src/com/cloud/storage/template/LocalTemplateDownloader.java]: What file can we have travis (during the test) upload? Do we make a dummy file?

Comment (DaanHoogland) [inline core/src/com/cloud/storage/template/LocalTemplateDownloader.java]: How about the standard template, or one of the mini machina from nux' site?

Comment (borisroman) [inline core/src/com/cloud/storage/template/LocalTemplateDownloader.java]: That would mean including one in the repository?

**Comment by wido** [comment]
Still looks good to me. pom.xml should be there

**Comment by bhaisaab** [comment]
LGTM

**Comment by DaanHoogland** [comment]
Looks like what I meant @borisroman , LgreatTM

**Pull request merged**